lib/http.js

```diff
@@ -76,13 +77,12 @@ bedrock.events.on('bedrock-express.configure.routes', app => {
     ensureAuthenticated,
     asyncHandler(async (req, res) => {
       const {account: {id: accountId}} = (req.user || {});
-      const {account, profile} = req.query;
-      if(!account) {
-        throw new BedrockError(
-          'The "account" query parameter must be specified.',
-          'DataError',
-          {account, httpStatusCode: 400, public: true});
-      }
+      let {account} = req.query;
+      const {profile} = req.query;
+
+      // if the `account` query parameter not specified, use session account
+      account = account || accountId;
```

Comment:
Why are we adding this feature? I won't die on this hill, but we had decided before not to do this.